### DISCUSSION TITLE
Built in Help in the GUI?

### DISCUSSION THREAD

**Discussion by decentropy** [cryptosharks131/lndg#79] [Ideas]
LNDg is hands down the best app for managing LND. Autopilot is awesome

If I could suggest any feature - it would simply be baking in help/tips into the UI.
There are so many options, I find it hard to recall what setting means what.

**Comment by uncle-jj** [comment]
Hey @decentropy you are welcome to join us if interested in improving the Ui. 

[Designthon](https://event.bitcoin.design/#project-recVLdKX09W2QAgL6) with the Bitcoin Design Comunity
[Legends of Lightning](https://makers.bolt.fun/story/introducing-lndgxyz--156) on Bolt.Fun

**Reply by cliffpeskin** [reply]
https://lndg.xyz/app/ looks awesome

**Comment by decentropy** [comment]
One (hopefully easy) suggestion I'd share is to use the "title" attribute in HTML throughout.

So when a new user is confused in the UI by fields like "AF-Multiplier", etc.... simply adding a short sentence in "title" attribute will show a mouseover text for users, helping them understand.

I think this simple addition, would go a long way to help users understand the powerful features of this app.
